plugin.video.catchuptvandmore/resources/lib/channels/tr/beyaz.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
the empty __init__.py file above is not needed with Python3. Please remove.

Reply:
fixed